lib/cache.js

```diff
@@ -1,8 +1,35 @@
-'use strict';
+'use strict'
 
-var LevelUp = require('level');
-var Path = require('path');
+const Winston = require('winston')
+const LevelUp = require('level')
+const Sublevel = require('level-sublevel')
+const Path = require('path')
 
-var path = Path.join(__dirname, '..', 'data');
+const path = Path.join(__dirname, '..', 'cache')
 
-module.exports = LevelUp(path);
+const leveldb = LevelUp(path)
+// main cache DB path
+const db = Sublevel(leveldb)
+
+exports.addSubLevelCacheDB = (levelKey) => {
+  return db.sublevel(levelKey)
+}
+
+exports.getSubLevelCacheDB = (levelKey, callback) => {
+  callback(db.sublevel(levelKey))
```

Comment:
This doesn't need a callback, it's synchronous. Also, I have the same question as https://github.com/jembi/facility-cache/pull/1/files#r101032418.